### ISSUE TITLE
[BUG]: A hardware accelerated encode of a complex scene flickers with out-of-order frames on DX11, works correctly on DX12

### ISSUE THREAD

**Issue by doctorpangloss** [Unity-Technologies/com.unity.webrtc#717]
### Package version

2.4.0-exp.7

### Environment

```markdown
* OS: Windows Server 20H2
* Unity version: 2021.2.15f1

NVIDIA Drivers: 515 Gaming
```

### Steps To Reproduce

1. Use a complex scene with screen space reflections, a reflection probe, and VFX graph.
2. Observe under DX11 the stream flickers with old frames, showing the stream out of order occasionally.
3. Observe under DX12 the stream works well, if at a slower FPS.

I observe the out of order frames issue on both my scenes with large, full screen VFX Graph assets. One of them is the Space Table scene from the VFX Graph samples. I suggest testing with that sample.

### Current Behavior

Out of order frames from the past flicker in and out of the stream in DX11.

### Expected Behavior

The stream should only go forward in time smoothly, like it does in DX12.

**Comment by karasusan** [comment]
@doctorpangloss 
Is it related the issue with VFX graph?

**Comment by karasusan** [comment]
memo: WRS-323

**Comment by karasusan** [comment]
fixed #722
